packages/core/src/components/description/description.scss

```diff
@@ -0,0 +1,14 @@
+@use '../../helpers';
+
+@mixin Description() {
+  .ods-description {
```

Comment:
Are you still considering changing the name to `.ods-helper-text`?

Reply:
I am yes, but will do as a separate PR.